### ISSUE TITLE
[Feature Request] Make isExpanded apply to root

### ISSUE THREAD

**Issue by bezko** [Lapple/react-json-inspector#30]
Is there a way to overide this behaviour, I guess it makes sens for most use case but for documents that have a lot of top level elements it would be nice to have the whole thing collapsed as an option.

https://github.com/Lapple/react-json-inspector/blob/34ca62490174b29568b3786bc9e7b1b808e09ccb/lib/leaf.js#L189

**Comment by keithkml** [comment]
Yep this was not the behavior I expected and I wasted some time trying to get it to hide the root on load.
